src/shared/libfido2-util.c

```diff
@@ -718,8 +718,8 @@ int fido2_generate_hmac_hash(
                 const void *user_id, size_t user_id_len,
                 const char *user_name,
                 const char *user_display_name,
-                const char *user_icon,
-                const char *askpw_icon,
+                const char *user_icon_name,
+                const char *askpw_icon_name,
```

Comment:
Why changed?

Reply:
Changed it according to the used argument comments, it seems you rather want it the other way around.